AcquisitionInfo/explainer.md

```diff
@@ -0,0 +1,223 @@
+  // The method must be recalled at a later time
+  return;
+}
+```
+
+### Accessing acquisition details
+
+Once we've accessed the provider, we can asynchronously retrieve a dictionary payload of
+acquisition details through the `getDetails()` method which returns a ScriptPromise object.
+
+```js
+details = await navigator.acquisitionInfoProvider.getDetails();
+let platform = details["storePlatform"]; // e.g. Microsoft Store
+let campaignId = details["campaignId"]; // e.g. adCampaign202306
+```
+
+While there are some common key-value pairs, depending on the platform of the acquisition the
+pairs contained within the dictionary may differ to fit different use cases.
+
+```js
+details = {
+  storePlatform: "Google Play Store",
+  installReferrer: "utm_source=google-play&utm_medium=organic",
+  referrerClickTimestampSeconds: 1623214800,
+  installBeginTimestampSeconds: 1623214810,
+  installBeginReferrer: "utm_source=google-play&utm_medium=referral&utm_campaign=campaign-123",
+  installVersion: "1.0.0",
+  googlePlayInstant: false
+}
+```
+
+```js
+details = {
+  storePlatform: "Apple App Store",
+  attribution: "true",
+  orgId: "40669820",
+  campaignId: "542370539",
+  conversionType: "Download",
+  adGroupId: "542317095",
+  countryOrRegion: "US",
+  keywordId: "87675432",
+  adId: "542317136"
+}
+```
+
+In the event that a web application has referral information that has expired for that specific
+application, or in any other case where acquisition info is missing for the web application,
+the API would return empty values for the associated properties in the dictionary. This is consistent with
+existing behavior in store information retrieval API for native applications.
+
+```js
+details = {
+  storePlatform: "Microsoft Store",
+  campaignId: ""
+}
+```
+
+## Attribution on browser installs
+
+While attribution is currently not being tracked in the same capacity as Store installations for browser-initiated
+installations of web applications, the Acquisition Info API could enable this sort of attribution capability.
+
+### Same-domain installation
+
+One solution for capturing attribution in organic same-domain installations initiated as an user agent (UA) install
+could be the use of query parameters. If the web application after installation
+has listed GET parameters in their URL as attribution information, within the API we can define acquisition related parameters
+to capture such as `c_id` and translate it into the dictionary return.
+
+```js
+// URL: foo.com?c_id=browserInstallationCampaign
+details = {
+  storePlatform: "",
+  campaignId: "browserInstallationCampaign"
+}
+```
+
+This could also be expanded upon to capture a set of acquisition parameters related to browser installation such as the web
+referrer or any parameter of interest that is already being captured in Store attribution.
+
+### Web Install API
+
+Currently UA installs are initiated by the user directly on the website they want to install.
```

Comment:
You're lacking sufficient detail here.  You've provided example payloads for all other types of installs but this one.  What does the payload for a web app installed from the Web install API look like?